.travis.yml

```diff
@@ -2,9 +2,15 @@ language: php
 php:
   - "5.4"
   - "5.5"
+  - "5.6"
+  - "nightly"
   - hhvm
 
 before_script:
   - composer install --dev
 
 script: ./vendor/bin/phpunit --coverage-text
+
+matrix:
+     allow_failures:
+         - php: hhvm  
```

Comment:
@nilopc hhvm was working fine before the PR, I would like to keep it compatible. Do you know why those tests are failing?